packages/components/src/components/input/input.stories.ts

```diff
@@ -1,932 +1,363 @@
-    { type: 'attribute', name: 'help-text', value: 'How would you like to be called?' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Label = {
+  render: () => html`
+    <div class="flex flex-row gap-12 items-baseline">
+      <sd-input label="Label Attribute" spellcheck></sd-input>
+      <sd-input spellcheck>
+        <div slot="label" class="text-lg">Label Slot</div>
+      </sd-input>
+    </div>
+  `
 };
 
 /**
- * Use the `placeholder` attribute to add a placeholder.
+ *Use the `placeholder` attribute to add a placeholder.
  */
-
-export const Placeholders = {
-  args: overrideArgs([{ type: 'attribute', name: 'placeholder', value: 'Type something' }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Placeholder = {
+  render: () => html`
+    <div class="w-[250px]">
+      <sd-input placeholder="Placeholder example" spellcheck></sd-input>
+    </div>
+  `
 };
 
 /**
- * Add the `clearable` attribute to add a clear button when the input has content.
+ * Use the `value` attribute to set the value.
  */
-
-export const Clearable = {
-  args: overrideArgs([{ type: 'attribute', name: 'clearable', value: true }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args
-        })}
-      </div>
-    `;
-  }
+export const Value = {
+  render: () =>
+    html` <div class="w-[250px]">
+      <sd-input value="Value example"></sd-input>
+    </div>`
 };
 
 /**
- * Add the `password-toggle` attribute to add a toggle button that will show the password when activated. Only works with `type="password"`.
+ * Use the `disabled` attribute to disable the input.
+ *
+ *  Use the `inputmode` attribute tells the browser what type of data will be entered by the user, allowing it to display the appropriate virtual keyboard on supportive devices.
+ * - `none`
+ * - `text`
+ * - `decimal`
+ * - `numeric`
+ * - `tel`
+ * - `search`
+ * - `email`
+ * - `url`
  */
 
-export const TogglePassword = {
-  name: 'Toggle Password',
-  parameters: {
-    controls: {
-      exclude: ['password-toggle']
-    }
-  },
-  args: overrideArgs([{ type: 'attribute', name: 'type', value: 'password' }]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          args,
-          constants: [
-            { type: 'attribute', name: 'password-toggle', value: true },
-            { type: 'attribute', name: 'label', value: 'Password' }
-          ]
-        })}
-      </div>
-    `;
-  }
+export const Disabled = {
+  render: () =>
+    html`<div class="w-[250px]">
+      <sd-input label="Label" value="Input text disabled" disabled></sd-input>
```

Comment:
```suggestion
      <sd-input label="Label" value="Disabled" disabled></sd-input>
```

Reply:
To align with what we have been doing in the other components. FYI @MartaPintoTeixeira @coraliefeil